index.test.js

```diff
@@ -0,0 +1,74 @@
+const http = require('http');
+const supertest = require('supertest');
+const handler = require('./src/handler');
+const parseEnv = require('./src/parse-env');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+
+const getExistingPosts = async () => {
+  const config = parseEnv(process.env);
+  const server = http.createServer(handler(config));
+
+  return (await supertest(server).get('/v0/Posts')).body;
```

Comment:
It feels weird to create a second server just to get some posts. Could we pass in an existing instance instead?

```js
const getExistingPosts = async (client) => {
  const response = await client.get('/v0/Posts');
  const body = response.body;

  return body.records;
};

const client = supertest(server);

it('...', async () => {
  const posts = await getExistingPosts(client);
  const post = posts[0];

  // ...
});
```

I'm not sure if this is possible with `superagent`, and may be related to my comment about `got`